bftclient/test/bft_client_api_tests.cpp

```diff
@@ -97,10 +97,13 @@ class ClientApiTestParametrizedFixture : public ClientApiTestFixture,
       else
         signature_data--;  // corrupt the signature
     }
-    bool sig_verification_result = transaction_verifier_->verify(reinterpret_cast<const char*>(request_data),
-                                                                 request_header->requestLength,
-                                                                 reinterpret_cast<const char*>(signature_data),
-                                                                 sig_len);
+    std::string data;
+    data.resize(request_header->requestLength);
+    std::string sig;
+    sig.resize(request_header->reqSignatureLength);
+    std::memcpy(data.data(), reinterpret_cast<const char*>(request_data), request_header->requestLength);
```

Comment:
Although it's a test, I think it's a bad practice to initialize a string by std::memcpy.
A better way is to use a constructor:
`std::string s{reinterpret_cast<const char*>(request_data), request_header->requestLength};`